.travis.yml

```diff
@@ -0,0 +1,36 @@
+os: linux
```

Comment:
why add Travis if you're already doing these tests in actions?

Reply:
Yeah I know, once I had prepared Travis CI I realized that it would be better to have Actions instead. Since Travis might not be for free in future. I will remove `travis.yml` once I finish migration to Actions.

Reply:
I didn't look through the commits to see the order of things. Makes sense.

Reply:
I think you're probably fine to go ahead and nuke this file and I can approve this PR and you can merge it. Once it's merged, then PRs will use the version in the PR, so you can do further development on it without merging... but you have to get the actions file merged before GH notices it for some reason.

Reply:
oh nvm, I already approved it. heh.